conf/application.conf

```diff
@@ -118,7 +118,7 @@ limits {
   maxDepartureMeansOfTransport = 999
   maxTransportEquipment = 9999
   maxAdditionalReferences = 99
-  maxDocuments = 99
+  maxDocumentsPerType = 99
```

Comment:
This can be overridden in the test application.conf so we're not working with unnecessarily large limits